tools/python/google_maps_bookmarks.py

```diff
@@ -104,26 +140,51 @@ def write_kml(self):
             point = ET.SubElement(placemark, "Point")
             ET.SubElement(point, "coordinates").text = place['coordinates']
         tree = ET.ElementTree(root)
-        tree.write(self.kml_file)
-        print()
+        tree.write(self.output_file)
         print()
-        print("Exported Google Saved Places to " + os.path.abspath(self.kml_file))
-
-
-def get_json(url):
-    max_attempts = 3
-    for retry in range(max_attempts):
-        try:
-            response = urllib.request.urlopen(url)
-            return json.load(response)
-        except urllib.error.URLError:
-            print(f"Couldn't connect to Google Maps. Retrying... ({retry + 1}/{max_attempts})")
-            if retry < max_attempts - 1:
-                continue
+        print("Exported Google Saved Places to " + path.abspath(self.output_file))
+
+    def write_gpx(self):
+        gpx = ET.Element("gpx", version="1.1", creator="GoogleMapsConverter")
+        for place in self.places:
+            wpt = ET.SubElement(gpx, "wpt", lat=place['coordinates'].split(',')[1], lon=place['coordinates'].split(',')[0])
+            ET.SubElement(wpt, "name").text = place['name']
+            ET.SubElement(wpt, "desc").text = place['description']
+        tree = ET.ElementTree(gpx)
+        tree.write(self.output_file)
+        print("Exported Google Saved Places to " + path.abspath(self.output_file))
```

Comment:
This line is duplicated in both `write_*` methods. You could move it at the end of the `convert` method.